system/lib/wasmfs/js_api.cpp

```diff
@@ -105,4 +105,21 @@ void _wasmfs_symlink(char* old_path, char* new_path) {
 long _wasmfs_chmod(char* path, mode_t mode) {
   return __syscall_chmod((intptr_t)path, mode);
 }
+
+// Helper method that identifies what a path is:
+//   ENOENT - if nothing exists there
+//   EISDIR - if it is a directory
+//   EEXIST - if it is a normal file
+long _wasmfs_identify(char* path) {
```

Comment:
Return int so this isn't 64-bit on wasm64

Reply:
(looks like the same should be done for other long-returning functions in this file).

Reply:
Good point, fixed.